recipes/sdl/all/conanfile.py

```diff
@@ -1,6 +1,11 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import is_apple_os
+from conan.tools.files import get, replace_in_file, rm, rmdir
+from conan.tools.files import patch as apply_patch
 from conan.tools.microsoft import is_msvc
-from conans import ConanFile, CMake, tools
-from conans.errors import ConanInvalidConfiguration
+from conan.tools.scm import Version
+from conans import CMake, tools
```

Comment:
```suggestion
from conans import CMake
```
No longer used

Reply:
it's still used: https://github.com/conan-io/conan-center-index/blob/6eaa24f5ca1c098cb6dd09d903d1407caa5f934b/recipes/sdl/all/conanfile.py#L358